tests/docker/languages/Dockerfile.testing.rust.big_endian.1_30_1

```
@@ -0,0 +1,15 @@
+FROM rust:1.30.1-slim-stretch as base
+
+WORKDIR /setup
+RUN apt -qq update >/dev/null
+RUN apt -qq install -y curl gcc-mips-linux-gnu
```

Need to install the `qemu-user` package so we can run MIPS binaries.

```suggestion
RUN apt -qq install -y curl gcc-mips-linux-gnu qemu-user
```